packages/flutter/test/widgets/media_query_test.dart

```diff
@@ -499,7 +499,7 @@ void main() {
     // values are copied over exactly
     const Size customSize = Size(3.14, 2.72);
     const double customDevicePixelRatio = 1.41;
-    const double customTextScaleFactor = 1.62;
+    const TextScaler customTextScaler = TextScaler.linear(1.23);
```

Comment:
Why did the scale change here?

Reply:
These are random values I think I just copied them over. Shouldn't really matter.